cmd/libs/go2idl/parser/parse.go

```diff
@@ -101,8 +107,18 @@ func (b *Builder) AddBuildTags(tags ...string) {
 // e.g. test files and files for other platforms-- there is quite a bit of
 // logic of that nature in the build package.
 func (b *Builder) buildPackage(pkgPath string) (*build.Package, error) {
+	// This is a bit of a hack.  The srcDir argument to Import() should
+	// properly be the dir of the file which depends on the package to be
+	// imported, so that vendoring can work properly.  We assume that there is
+	// only one level of vendoring, and that the CWD is inside the GOPATH, so
```

Comment:
Pretty sure this will break OpenShift and any third party resource groups people create outside of the main kube tree (since they'll have to vendor).  I'll take a look to see what happens.